test/regression.uts

```diff
@@ -517,6 +517,52 @@ assert(conf.manufdb._resolve_MAC("00:00:0F:01:02:03") == "Next:01:02:03")
 assert(conf.manufdb._get_short_manuf("00:00:0F:01:02:03") == "Next")
 assert(in6_addrtovendor("fe80::0200:0fff:fe01:0203").lower().startswith("next"))
 
+assert conf.manufdb.lookup("00:00:0F:01:02:03") == ('Next', 'Next, Inc.')
+assert "00:00:0F" in conf.manufdb.reverse_lookup("Next")
+
+= Test multiple wireshark's manuf formats
+~ manufdb
+
+old_format = """
+# comment
+00:00:00    JokyIsland    Joky Insland Corp SA
+00:01:12    SecdevCorp    Secdev Corporation SA LLC
+EE:05:01    Scapy         Scapy CO LTD & CIE
+FF:00:11    NoName
+"""
+new_format = """
+# comment
+00:00:00    JokyIsland  #  Joky Insland Corp SA
+00:01:12    SecdevCorp  #  Secdev Corporation SA LLC
+EE:05:01    Scapy       #  Scapy CO LTD & CIE
+FF:00:11    NoName
+"""
```

Comment:
Hi, this commit looks good to me, but the variable names here seem the be the other way round: the old format of `manuf` uses `OUI short # long` while the new one is `OUI short long`.

Reply:
Nice catch. I switched the var names